packages/web/src/scss/tools/_responsive-properties.scss

```diff
@@ -0,0 +1,84 @@
+// 1. Internal variables:
+//
+//    a) Internal custom property the input custom properties are applied to.
```

Comment:
**nitpick (non-blocking):** 
```suggestion
//    a) Internal custom property the input's custom properties are applied to.
```

I did not understand the comment. Maybe it should be like I suggested. 🤔

Reply:
It has nothing to do with inputs, but I tried to improve it 🙂.